core/src/main/java/io/grpc/internal/GrpcUtil.java

```diff
@@ -131,9 +131,9 @@
   public static final String MESSAGE_ACCEPT_ENCODING = "grpc-accept-encoding";
 
   /**
-   * The default maximum uncompressed size (in bytes) for inbound messages. Defaults to 100 MiB.
+   * The default maximum uncompressed size (in bytes) for inbound messages. Defaults to 4 MiB.
```

Comment:
Ah, okay. I added something to the builders, since those are what are actually observed by users.